src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -197,21 +200,24 @@ private static StudyInfos toInfos(StudyEntity entity) {
                 .build();
     }
 
-    private static BasicStudyInfos toBasicInfos(BasicStudyEntity entity) {
+    private static BasicStudyInfos toBasicInfos(StudyCreationRequestEntity entity) {
         return BasicStudyInfos.builder().studyName(entity.getStudyName())
-                .creationDate(ZonedDateTime.ofInstant(entity.getDate().toInstant(ZoneOffset.UTC), ZoneId.of("UTC")))
```

Comment:
why change the storage from localdatetime to zoneddatetime ? I think it's a good idea but it has some implications. Maybe do this in a separate PR ?